lib/foreman_tasks_core/continuous_output.rb

```diff
@@ -1,50 +0,0 @@
-module ForemanTasksCore
-  class ContinuousOutput
```

Comment:
Didn't we want to keep this?
Maybe transfering to a `ForemanTasks` namespace? :thinking: